packages/core/src/sass/app/_common/components/wallet.scss

```diff
@@ -0,0 +1,29 @@
+$wallet-bg-color: (
+    'usd': (
+        'light': linear-gradient(to right, #f2bfba, #e8b4c0, #d6acc7, #bca7cc, #9ca3cc),
+        'dark': linear-gradient(to right, #652723, #602033, #531f41, #3d234b, #1e274f),
+    ),
+    'eur': (
+        'light': linear-gradient(to right, #b5bbd9, #e0bcde, #ffc0ca, #ffcfad, #fce99f),
+        'dark': linear-gradient(to right, #1c2444, #4f284b, #76313f, #864a2a, #7c6c21),
+    ),
+    'aud': (
+        'light': linear-gradient(to right, #a9e5ba, #b9e8ad, #cdea9f, #e5ea93, #ffe88a),
+        'dark': linear-gradient(to right, #214f3e, #2e5937, #44622a, #60681b, #816b0d),
+    ),
+);
+
```

Comment:
@hamid-deriv add a comment here how to use this mixin in the class

Reply:
This mixin is initialized in this file, so there is no need to use this mixin in the style, it will generate classes for bg like this:

`.wallet__usd-bg`
`.wallet__usd-bg--dark`

Also, I add the sample output class in the file as a comment, too.